lib/practice.md

```diff
@@ -3,58 +3,59 @@
 - What package did you install for enumeration?
 
 ```md
-<!-- your answer here -->
+enum
+npm install --save enum
 ```
 
 ## Research: Popular NPM Packages
 
 Package 1:
 ```md
-<!-- your answer here -->
+moment
 ```
 1.  What problem does this package solve?
 ```md
-<!-- your answer here -->
+Working with dates
 ```
 1.  How can I use this package in my own code? What steps are involved?
 ```md
-<!-- your answer here -->
+I can use when parsing and validating dates
 ```
 1.  How well-maintained is this package? What criteria did I use to decide?
 ```md
-<!-- your answer here -->
+Very well maintained - 3,242 commits and 60 releases on gitHub.
 ```
 
 Package 2:
 ```md
-<!-- your answer here -->
+grunt
 ```
 1.  What problem does this package solve?
 ```md
-<!-- your answer here -->
+It allows a developer to do less repetative tasks with testing (and other tasks).  It also offers a number of plugins
 ```
 1.  How can I use this package in my own code? What steps are involved?
 ```md
-<!-- your answer here -->
+Creating tests for my code
```

Comment:
This package doesn't create tests.  Check out [Mocha](https://mochajs.org) if you're interested in trying out a testing framework.